Formula/imagemagick.rb

```diff
@@ -132,6 +136,14 @@ def caveats
   end
 
   test do
-    system "#{bin}/identify", test_fixtures("test.png")
+    system bin/"identify", test_fixtures("test.png")
+    # Test recommended features and delegates.
+    features = shell_output("#{bin}/convert -version")
+    assert_match "freetype", features if build.with? "freetype"
```

Comment:
We do not, generally, no but if you're adding `build.with?` it's probably worth adding the inverse.